CHANGELOG.md

```diff
@@ -37,6 +37,8 @@ and this project adheres to
   - [#4037](https://github.com/bpftrace/bpftrace/pull/4037)
 - Add warning when unset or empty positional parameters are used
   - [#4095](https://github.com/bpftrace/bpftrace/pull/4095)
+- Support access arg0-arg11 in USDT probes
```

Comment:
From the comment/code below, it looks like we can support up to arg255. Also, I might re-word this to say "Support accessing up to 255 USDT probe args"